tools/ci_build/github/linux/docker/scripts/install_ubuntu.sh

```diff
@@ -77,3 +78,5 @@ fi
 /usr/bin/python${PYTHON_VER} -m pip install --upgrade --force-reinstall requests==2.21.0
 rm -rf /var/lib/apt/lists/*
 
+aria2c -q -d /tmp -o llvm.tar.xz https://github.com/llvm/llvm-project/releases/download/llvmorg-8.0.1/clang+llvm-8.0.1-x86_64-linux-gnu-ubuntu-14.04.tar.xz
+tar --strip 1 -Jxf /tmp/llvm.tar.xz -C /usr
```

Comment:
Linux only?

Reply:
I'd prefer to not create a pipeline and docker file. We already have too many build pipelines, even more than CI build machines we have.